docs/modules/ROOT/pages/documentation/extensions/grpc/README.adoc

```diff
@@ -0,0 +1,401 @@
+              <groupId>kr.motd.maven</groupId>
+              <artifactId>os-maven-plugin</artifactId>
+              <version>1.6.2</version>
+          </extension>
+      </extensions>
+      <plugins>
+          <plugin>
+              <groupId>org.xolstice.maven.plugins</groupId>
+              <artifactId>protobuf-maven-plugin</artifactId>
+              <version>0.6.1</version>
+              <configuration>
+                  <protocArtifact>com.google.protobuf:protoc:3.19.2:exe:${os.detected.classifier}</protocArtifact>
+                  <pluginId>grpc-java</pluginId>
+                  <pluginArtifact>io.grpc:protoc-gen-grpc-java:1.43.1:exe:${os.detected.classifier}</pluginArtifact>
+              </configuration>
+              <executions>
+                  <execution>
+                      <goals>
+                          <goal>compile</goal>
+                          <goal>compile-custom</goal>
+                      </goals>
+                  </execution>
+              </executions>
+          </plugin>
+      </plugins>
+  </build>
+----
+
+[[service-implementation]]
+=== Service Implementation
+
+After doing the previous step, you can build the project to generate the stubs. 
+
+Example:
+[source, shell]
+----
+${webapp root folder} > mvn clean install
+----
+
+The stubs should be generated with similar structure like the following:
+
+* target
+** generated-sources
+*** protobuf
+**** grpc-java
+***** fish.payara.samples.grpc (*here you can see the service class*)
+**** java
+***** fish.payara.samples.grpc (*here you can see the types associated to the service*)
+
+Then you must create a class to extend the internal generated class with ImplBase particle. In our case, this class is located into PayaraServiceGrpc under the folder `/target/generated-sources/protobuf/grpc-java` .
+
+[[service-code-implementation]]
+==== Service Code Implementation
+
+[source, Java]
+----
+@Dependent
+public class PayaraService extends PayaraServiceGrpc.PayaraServiceImplBase {
+    private final static Logger log = Logger.getLogger(PayaraService.class.getName());
+    @Override
+    public void communicate(fish.payara.samples.grpc.PayaraReq request,
+                            io.grpc.stub.StreamObserver<fish.payara.samples.grpc.PayaraResp> responseObserver) {
+        final String message = request.getMessage();
+        log.info(String.format("Processing message: %s", message));
+        responseObserver.onNext(response(message));
+        responseObserver.onCompleted();
+    }
+
+    private static final fish.payara.samples.grpc.PayaraResp response(String message) {
+        return fish.payara.samples.grpc.PayaraResp.newBuilder() 
+                .setMessage(message) 
+                .build();
+    }
+}
+----
+
+Our example is a simple echo service that will print the HelloWorld message. On the next section when creating the client you will see more details.
+
+Finall step is to deploy the application to the server. 
```

Comment:
"The final step is to deploy the application in a Payara Server domain"

Reply:
Also, does the server print out any details of the gRPC services deployed in tandem with the application? It might be good to give a sample of the log entries printed out by the server to compliment this step.

Reply:
for now it was not added any to print in logs to indicate service deployed